Sources/RxFeedback/ObservableType+RxFeedback.swift

```diff
@@ -97,7 +97,7 @@ extension SharedSequenceConvertibleType where E == Any, SharingStrategy == Drive
                 initialState: initialState,
                 reduce: reduce,
                 scheduler: SharingStrategy.scheduler,
-                scheduledFeedback: observableFeedbacks
+                feedback: observableFeedbacks
```

Comment:
This is a breaking a change, right?

Reply:
We have 1.0 already

Reply:
@sergdort What do you mean with we have 1.0 ??

Reply:
I mean that some time ago we released `RxFeedback` 1.0. So we need to consider if it worth to make a `2.0` just because of parameter rename :)

Reply:
Uhmm got it @sergdort !!! You're right